flink-python/pyflink/common/typeinfo.py

```diff
@@ -48,30 +48,12 @@ class acts as the tool to generate serializers and comparators, and to perform s
     nested types).
     """
 
-
-class WrapperTypeInfo(TypeInformation):
-    """
-    A wrapper class for java TypeInformation Objects.
-    """
-
-    def __init__(self, j_typeinfo):
-        self._j_typeinfo = j_typeinfo
+    def __init__(self):
+        self._j_typeinfo = None
 
     def get_java_type_info(self) -> JavaObject:
```

Comment:
Since the subclasses of `TypeInformation` will always override this method, I think this method can be declared as an abstract method.